src/main/java/org/nantipov/utils/wordhugopress/config/job/convertposts/ConvertPostsReaderConfig.java

```diff
@@ -112,7 +83,39 @@ public ConvertPostsReaderConfig(SourcesSettings sourcesSettings) {
                 -1
         );
     }
-
+    
+    private String getSQLQueryPost(String TablePrefix){
```

Comment:
1. I would name rename `getSQLQueryPost()` to `getSQLQueryPosts()` to prevent former naming (query returns multiple posts).
2. Java naming convention. Rename `TablePrefix` -> `tablePrefix` (non-capitalized camel case).